### ISSUE TITLE
Error with spaces between params of function in assembly block

### ISSUE THREAD

**Issue by PaulRBerg** [sc-forks/solidity-coverage#317]
When covering a function that is written in inline assembly, which has spaces between its parameters, the following error is thrown:

> SyntaxError: Expected identifier but " " found.

Example code:

```solidity
function myFunction(note, k, a)
```

If I remove the sapces, like below, everything's fine:

```solidity
function myFunction(note,k,a)
```

**Comment by area** [comment]
I don't believe we've ever instrumented assembly (as you can't emit events, as far as I know), but I agree that it shouldn't balk when presented with valid assembly. From the small test case I constructed, I believe the `leapdao` branch should work. If not, please supply a valid contract that fails (of any size, doesn't have to be a minimum valid example). Note that branch only supports Solidity ^0.5.1

**Comment by PaulRBerg** [comment]
Hey there, thanks for getting back and for your latest work on v5. Unfortunately, I was not able to reproduce this in the latest version of [AZTEC][aztec-monorepo], but I'll post here if I do. I'm certain that the error exists because I had to manually remove the space between the function parameters.

[aztec-monorepo]: https://github.com/AztecProtocol/AZTEC

**Comment by area** [comment]
I actually can't find an example in that codebase of a function being declared in assembly. Either I'm looking for the wrong syntax, or my `grep` skills need polishing - can you point me to one?

**Comment by PaulRBerg** [comment]
Sure, so if you checkout the `feat-coverage` branch you will find the contracts in `/packages/protocol`. We switched to the @0x/sol-coverage there, but there's also a recent commit where solidity-coverage is being used.

**Comment by area** [comment]
Ah, you were using the leapdao fork, which uses the old (custom) parser, which is why I was not able to reproduce with the leapdao branch that I am working on in #318 (which is based on the leapdao fork, but with extra changes, including a stock parser. If is unfortunately confusingly named for this conversation...).

Unfortunately, the new parser balks at your contracts for a different reason. I've raised an issue up the chain: https://github.com/federicobond/solidity-parser-antlr/issues/53 if interested.

**Comment by cgewecke** [comment]
Fixed in the @beta and `0.6.0`.
